tests/unit/runtimes/node/in_source_config.test.ts

```diff
@@ -262,7 +262,7 @@ describe('V2 API', () => {
     })
 
     test('With an invalid pattern', () => {
-      expect.assertions(4)
+      expect.assertions(8)
```

Comment:
oof 🫣

Reply:
I didn't think it was worth splitting into another test.

Reply:
I've split this into two tests now.